src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -147,7 +147,7 @@ configurationRegistry.registerConfiguration({
 				CreateNewTerminalAction.ID,
 				CopyTerminalSelectionAction.ID,
 				KillTerminalAction.ID,
-				FocusTerminalAction.ID,
+				FocusActiveTerminalAction.ID,
```

Comment:
Yes they should, if not the terminal may eat the keystrokes before they reach the vscode keybinding system.